packages/playwright-core/src/client/elementHandle.ts

```diff
@@ -270,6 +270,8 @@ export async function convertInputFiles(files: string | FilePayload | string[] |
     if (!items.every(item => typeof item === 'string'))
       throw new Error('File paths cannot be mixed with buffers');
     if (context._connection.isRemote()) {
+      if (!context._isLocalBrowserOnServer())
+        throw new Error('File paths are not supported in this type of browser, please set buffers instead');
```

Comment:
I thought we can read small files and transport them as buffers here? Same as line 286.